### PR TITLE
Add new param on_off_invert.

### PR DESCRIPTION
**Pull request by tombbo**
Some KNX Room controller (RCD 20xx, 2178) have inverted power ON/OFF value. ON means disable controller, OFF means enable controller.

### CODE REVIEW

**Comment by farmio** [comment]
Climate.on is a RemoteValueSwitch which has an attribute `invert` that defaults to false. I'd use this instead of changing the implementation of Climate.turn_on() to `self.on.off()`. No   need to touch Climate.is_on property then and better readability.

**Comment by tombbo** [comment]
Changed to use an attribute invert of a RemoteValueSwitch.

File: xknx/devices/climate.py [hunk 1]
```diff
@@ -83,7 +83,8 @@ def __init__(self,
                  min_temp=None,
                  max_temp=None,
                  mode=None,
-                 device_updated_cb=None):
+                 device_updated_cb=None,
+                 on_off_inverted=False):
```
Comment (farmio) [inline xknx/devices/climate.py]: Please move it to line 83 just after `group_address_on_off_state=None,` to have all on_off things together.

File: xknx/devices/climate.py [hunk 2]
```diff
@@ -130,7 +131,8 @@ def __init__(self,
             group_address_on_off,
             group_address_on_off_state,
             device_name=self.name,
-            after_update_cb=self.after_update)
+            after_update_cb=self.after_update,
+            invert=on_off_inverted)
```
Comment (farmio) [inline xknx/devices/climate.py]: Consider renaming it to `on_off_invert` to be consistent with the RemoteValueSwitch and Cover attributes.

**Comment by tombbo** [comment]
Done.

**Pull request merged**